### PR TITLE
Interview goal

### PR DESCRIPTION
**Pull request by ArnaudChirat**

### CODE REVIEW

File: interview/views.py [hunk 1]
```diff
@@ -471,7 +474,17 @@ def minute(request, interview_id, slug_info=None):
     except Interview.DoesNotExist:
         return HttpResponseNotFound()
 
-    context = {"interview": interview, "process": interview.process, "subsidiaries": Subsidiary.objects.all()}
+    next_goal = interview
+    if not next_goal.next_interview_goal:
+        process = interview.process
+        next_goal = Interview.objects.filter(process=process).exclude(next_interview_goal="").last()
```
Comment (bport) [inline interview/views.py:480]: Next goal need to be current interview goal or previous interview next goal.
We need to create a new database field to handle that

File: interview/views.py [hunk 2]
```diff
@@ -471,7 +471,21 @@ def minute(request, interview_id, slug_info=None):
...
         return HttpResponseNotFound()
 
-    context = {"interview": interview, "process": interview.process, "subsidiaries": Subsidiary.objects.all()}
+    try:
+        goal = (
+            interview.goal
+            if interview.goal
+            else Interview.objects.filter(process=interview.process).reverse()[1].next_interview_goal
+        )
+    except:
```
Comment (bport) [inline interview/views.py:480]: Catch only needed exception

Comment (bport) [reply interview/views.py:480]: I will avoid inline if else when we are not in the same line

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -241,35 +241,42 @@ msgid "Planned date"
...
 
 #: interview/models.py:469
-#: interview/templates/interview/interview_minute.html:31
+#: interview/templates/interview/interview_minute.html:36
 msgid "Next interview goal"
 msgstr "Objectif du prochain entretien"
 
 #: interview/models.py:470
+#: interview/templates/interview/interview_minute.html:18
+#, fuzzy
```
Comment (bport) [inline locale/fr/LC_MESSAGES/django.po:261]: this line is marked as fuzzy fix it

**Review by bport — CHANGES_REQUESTED** [review]
Perhaps on new interview form, we can display next goal somewhere and add placeholder to explain that we need to enter a goal only if goal differs

**Comment by ArnaudChirat** [comment]
> Perhaps on new interview form, we can display next goal somewhere and add placeholder to explain that we need to enter a goal only if goal differs

**Comment by ArnaudChirat** [comment]
> Perhaps on new interview form, we can display next goal somewhere and add placeholder to explain that we need to enter a goal only if goal differs

May be we can preload the form with next_goal

File: interview/views.py [hunk 4]
```diff
@@ -517,7 +519,20 @@ def minute(request, interview_id, slug_info=None):
     except Interview.DoesNotExist:
         return HttpResponseNotFound()
 
-    context = {"interview": interview, "process": interview.process, "subsidiaries": Subsidiary.objects.all()}
+    try:
+        if interview.goal:
+            goal = interview.goal
+        else:
+            goal = Interview.objects.filter(process=interview.process).reverse()[2].next_interview_goal
```
Comment (bport) [inline interview/views.py:526]: I think we can use rank to get previous one

**Review by bport — CHANGES_REQUESTED** [review]

**Pull request merged**